docs/ops.html

```diff
@@ -4498,9 +4498,17 @@ <h4 class="anchor-heading"><a id="eligible_leader_replicas_upgrade" class="ancho
   <p>Downgrades are safe to perform by setting <code>eligible.leader.replicas.version=0</code>.</p>
 
   <h4 class="anchor-heading"><a id="eligible_leader_replicas_tool" class="anchor-link"></a><a href="#eligible_leader_replicas_tool">Tool</a></h4>
-  <p>The ELR fields can be checked through the API DescribeTopicPartitions. The admin client can fetch the ELR info by describing the topics.
-    Also note that, if <code>min.insync.replicas</code> is updated for a topic, the ELR field will be cleaned. If cluster default min ISR is updated,
-    all the ELR fields will be cleaned.</p>
+  <p>The ELR fields can be checked through the API DescribeTopicPartitions. The admin client can fetch the ELR info by describing the topics.</p>
+  <p>Note that when the ELR feature is enabled:</p>
+  <ul>
+    <li>The cluster-level <code>min.insync.replicas</code> config will be added if there is not any. The value is the same as the static config in the active controller.</li>
+    <li>The previously set <code>min.insync.replicas</code> value at the broker-level config will be removed.</li>
+    <li>The override of <code>min.insync.replicas</code> in broker-level will be removed.</li>
+    <li>The alteration of <code>min.insync.replicas</code> config in broker-level is not allowed.</li>
+    <li>The removal of <code>min.insync.replicas</code> config in cluster-level is not allowed.</li>
+    <li>If <code>min.insync.replicas</code> is updated for a topic, the ELR field will be cleaned.</li>
+    <li>If the cluster default <code>min.insync.replicas</code> is updated, all the ELR fields will be cleaned.</li>
```

Comment:
nit: A `ClearElrRecord` is created even when the `min.insync.replicas` value remains unchanged. Perhaps we could either document the behavior or consider skipping the record generation in such case.